src/opnsense/scripts/unbound/blocklists/default_bl.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/local/bin/python3
+
+"""
+    Copyright (c) 2023 Deciso B.V.
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    1. Redistributions of source code must retain the above copyright notice,
+     this list of conditions and the following disclaimer.
+
+    2. Redistributions in binary form must reproduce the above copyright
+     notice, this list of conditions and the following disclaimer in the
+     documentation and/or other materials provided with the distribution.
+
+    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+"""
+
+import syslog
+import re
+import os
+import ujson
+from . import BaseBlocklistHandler
+
+class DefaultBlocklistHandler(BaseBlocklistHandler):
+    def __init__(self):
+        super().__init__('/tmp/unbound-blocklists.conf')
+        self.priority = 100
+        self._whitelist_pattern = self._get_excludes()
+
+    def get_config(self):
+        cfg = {}
+        if self.cnf and self.cnf.has_section('settings'):
+            if self.cnf.has_option('settings', 'address'):
+                cfg['dst_addr'] = self.cnf.get('settings', 'address')
+            if self.cnf.has_option('settings', 'rcode'):
+                cfg['rcode'] = self.cnf.get('settings', 'rcode')
+        return cfg
+
+    def get_blocklist(self):
+        result = {}
+        for blocklist, bl_shortcode in self._blocklists_in_config():
+            per_file_stats = {'uri': blocklist, 'skip': 0, 'blocklist': 0, 'lines': 0}
+            for entry in self._domains_in_blocklist(blocklist):
+                domain = entry.lower()
```

Comment:
```
per_file_stats['lines'] += 1
```
^^ missing (always reporting 0), but `_domains_in_blocklist()` already cleanses the data, it might be less relevant here to report.

Reply:
Nice catch, moved it to `_blocklist_reader()` as a separate log line.